api/src/main/java/io/minio/PostPolicy.java

```diff
@@ -167,14 +167,6 @@ public String bucketName() {
   }
 
   /**
-   * Returns form data of this post policy.
-   */
-  public Map<String,String> formData(String accessKey, String secretKey)
```

Comment:
> Removal makes backward incompatible. Is it expected?

You are right, PostPolicy is really used by the developer. However, this function doesn't seem to be working properly before, as it uses "us-east-1" by default unless another API was called before and updated the region cache with the correct value.